src/modules/prepareSpecificSettingToSave.js

```diff
@@ -71,14 +68,10 @@ export const prepareSpecificSettingsToSave = ({
                     )
                 }
 
-                summarySettings =
-                    (states.specificSetting.eventsDownload
-                        ? states.specificSetting.eventsDownload
-                        : specificSettingsDefault.eventsDownload) +
-                    ' events per OU'
+                summarySettings = `${states.specificSetting.eventsDownload} events per OU`
```

Comment:
Do these need to be translated with i18n?

Reply:
I'm translating this line as well as line 53 and 83